index.js

```diff
@@ -28,18 +28,24 @@
 
   'use strict';
 
+  var util = {inspect: {}};
+
   /* istanbul ignore else */
   if (typeof module === 'object' && typeof module.exports === 'object') {
-    module.exports = f (require ('sanctuary-show'),
+    module.exports = f (require ('util'),
+                        require ('sanctuary-show'),
                         require ('sanctuary-type-classes'));
   } else if (typeof define === 'function' && define.amd != null) {
-    define (['sanctuary-show', 'sanctuary-type-classes'], f);
+    var name = 'sanctuary-descending#util';
+    define (name, [], function() { return util; });
+    define ([name, 'sanctuary-show', 'sanctuary-type-classes'], f);
```

Comment:
Alternatively to defining a ad-hoc name-spaced module, you could probably wrap the definition function:

```js
define(['sanctuary-show', 'sanctuary-type-classes'], function(show, Z) {
  return f (util, show, Z);
});
```

Reply:
Good idea! I prefer this, despite the minor downside of having to forward the arguments.